libraries/app/application.cpp

```diff
@@ -1073,6 +1073,13 @@ std::shared_ptr<abstract_plugin> application::get_plugin(const string& name) con
    return my->_active_plugins[name];
 }
 
+bool application::is_plugin_enabled(const string& name) const
+{
+   if(my->_active_plugins.find(name) == my->_active_plugins.end())
```

Comment:
`if (x) return false; else return true;` can be simplified to `return !x;`.

Reply:
https://github.com/bitshares/bitshares-core/pull/1725/commits/26e9f16b3f5f88cd186ec3c853aa7278d73c7419